packages/fx-core/src/plugins/resource/cicd/providers/provider.ts

```diff
@@ -49,15 +49,15 @@ export class CICDProvider {
         this.providerName,
         "README.md"
       );
-      const readmeContent = await this.readLocalFile(localReadMePath);
+
       try {
-        await fs.writeFile(targetReadMePath, readmeContent);
+        await fs.copyFile(localReadMePath, targetReadMePath);
```

Comment:
According to the [document](https://nodejs.org/api/fs.html#fspromisescopyfilesrc-dest-mode). By default, targetReadMePath is overwritten if it already exists. Do we expect it?

Reply:
Yeah, there's existence check before, so it doesn't influence anything here.